frontend/src/lib/utils.tsx

```diff
@@ -1827,3 +1828,13 @@ export function shouldCancelQuery(error: any): boolean {
     // the query will continue running in ClickHouse
     return error.name === 'AbortError' || error.message?.name === 'AbortError' || error.status === 504
 }
+/**
+ * Some kea logics are used heavily across multiple areas so we keep it mounted once loaded with this trick.
+ */
+export function permanentlyMountLogic(logic: LogicWrapper): void {
+    const mountedLogic = logic.findMounted()
+    if (mountedLogic && !mountedLogic.cache._permanentMount) {
+        mountedLogic.cache._permanentMount = true
+        logic.mount()
+    }
+}
```

Comment:
You could take the abstraction one step further, and build something like this (pseudocode, didn't run)

```ts
function permanentlyMount(builtLogic: BuiltLogic): void {
    afterMount(() => {
        if (!builtLogic.cache._permanentMount) {
            builtLogic.cache._permanentMount = true
            builtLogic.wrapper.mount()
        }
    )(logic)
}
```

Then you can just add:

```ts
const actionsModel = kea([
    actions(...),
    reducers(...),
    permanentlyMount,
    // other stuff
])
```

... without needing to ask the user to put it into an `afterMount` effect themselves.

See here for docs on "[logic builders](https://keajs.org/docs/meta/kea#logic-builders)"

Reply:
Ah thats really nice. didnt even think about that 👍